<?xml version="1.0"?>
<!--
-->
<launch>

	<node name="marker_navigation" 
		pkg="marker_navigation" 
		type="marker_navigation_node" 
		respawn="false" 
		output="screen">
		
		<param name="pose_markers" value="/ar_pose_marker"/>		


		<param name="verbose" type="bool" value="true"/>
		
	</node>

</launch>
